.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,55 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.183.0/containers/dotnet
```

Comment:
why do you base it on the dotnet template? (also commented out configs further below)

Would the python3 template make more sense?
https://github.com/microsoft/vscode-dev-containers/tree/v0.183.0/containers/python-3

Reply:
I just used an image to let me debug.
Please, feel free to change it to anything more appropriate!